### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by dbereket**
Added the following changes:
- Fixed some typos w/ the React components (`DataView` was accidentally called `ButtonView`) & the scope argument in `ButtonView`
- Attempted to wrap the the GET request to Spotify in a try/catch loop (for debugging purposes in the console)
- Created a `query_constants` variable in case we need to reference `client_id` and `redirect_uri` somewhere else (tbh, this might be unnecessary at this stage -- but at some point it might make sense to stick these in a .env file or whatever implementation works best w/ react)

### CODE REVIEW

File: app/src/components/ButtonView.tsx [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
 const ButtonView: React.FC = () => {
     const scopes = ['user-top-read', 'user-read-recently-played', 'user-read-private', 'playlist-read-private', 'playlist-read-collaborative'];
 
     const query = {
-        scopes: scopes.join(' '),
-        redirect_uri: 'http://localhost:3000/data',
-        client_id: 'b711a8f79fce454f9b2352b78f7b07e9',
+        scope: scopes.join(' '),
+        redirect_uri: query_constants['redirect_uri'],
```
Comment (mdschramm) [inline app/src/components/ButtonView.tsx]:
```suggestion
        redirect_uri: query_constants.redirect_uri,
```

File: app/src/components/ButtonView.tsx [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+        client_id: query_constants['client_id'],
```
Comment (mdschramm) [inline app/src/components/ButtonView.tsx]:
```suggestion
        client_id: query_constants.client_id,
```

File: app/src/components/ButtonView.tsx [hunk 3]
```diff
@@ -1,15 +1,18 @@
 import { stringify } from "query-string";
 import React from 'react';
 
-
+const query_constants = {
```
Comment (mdschramm) [inline app/src/components/ButtonView.tsx]: I like the constants, added a minor style suggestion below, using dots to access properties is generally the js way. I see your python background here haha

Comment (dbereket) [reply app/src/components/ButtonView.tsx]: Lol yeah still getting used to js

**Pull request merged**